src/modules/ipfix/aggregator/AggregatorBaseCfg.cpp

```diff
@@ -140,6 +141,9 @@ Rule::Field* AggregatorBaseCfg::readNonFlowKeyRule(XMLElement* e)
 	ruleField->semantic = ie.getSemantic();
 	ruleField->fieldIe = ie.getFieldIe();
 
+	if (!BaseHashtable::isToBeAggregated(ruleField->type))
```

Comment:
Can you add braces around the then-part of the `if` statement?

Reply:
I'm used to kernel style. :-) Added the braces.